.github/workflows/scala3-build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           -Dmultinode.Xms256M \
           -Dmultinode.Xmx256M \
           -Dmultinode.XX:+AlwaysActAsServerClassMachine \
-          "+~ 3 ${{ matrix.command }}"
+          "+~ 3.3 ${{ matrix.command }}"
```

Comment:
can we use `++3.3` instead? I think the `+~` is no longer needed because sbt `++` doesn't need the full version any more

Reply:
Will update